styleguide/components/41-organisms/programme/_programme.scss

```diff
@@ -0,0 +1,194 @@
+.programme {
+  width: calc(100% + 2 * #{$gutter-width});
+  margin-right: -$gutter-width;
```

Comment:
@delrueba Don't we have a mixin for this behaviour?

Reply:
Can't find one on first sight 🔍